rllib/evaluation/rollout_worker.py

```diff
@@ -1258,6 +1265,8 @@ def add_policy(
                 )
             }
 
+        connectors_enabled = merged_config.get("enable_connectors", False)
```

Comment:
can you move this down to right above where it's first used?

Reply:
Done.